src/mongocrypt-endpoint-private.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MONGOCRYPT_ENDPOINT_PRIVATE_H
+#define MONGOCRYPT_ENDPOINT_PRIVATE_H
+
+#include <bson/bson.h>
```

Comment:
I'm sure I'm missing something but everything here seems to be from either `mongocrypt-status-private.h` or the standard library. Why import this?

Reply:
Hah you're not missing anything! It sure isn't needed. Removed.